prototype/link/scripts/explore_motion_firstlevel.sh

```diff
@@ -0,0 +1,198 @@
+#!/bin/bash
```

Comment:
This is more of a niche application for your study, rather than something that should go in the main neuropipe. This script is wrapping a bunch of other tools that already allow a user to do something like this